### PR TITLE
Add up date picker for the release date input

### PR DESCRIPTION
**Pull request by Agent-Ebimene**

### CODE REVIEW

File: api/comicIssue/queries/useUpdateComicIssue.ts [hunk 1]
```diff
@@ -13,12 +13,15 @@ const updateComicIssue = async (id: string | number, request: UpdateComicIssueDa
 
 export const useUpdateComicIssue = (id: string | number) => {
 	const toaster = useToaster()
+	const queryClient = useQueryClient()
 
 	return useMutation({
 		mutationFn: (request: UpdateComicIssueData) => updateComicIssue(id, request),
 		onSuccess: () => {
-			toaster.add('Comic Issue updated!', 'success')
+			toaster.add('Comic Issue updated!', 'success'),
```
Comment (luka-fundl) [inline api/comicIssue/queries/useUpdateComicIssue.ts]: lint, remove comma

**Comment by luka-fundl** [comment]
@Agent-Ebimene it looks okay to me, but maybe let's do few changes here:
1. Place release date field above authors field (create and edit screens)
2. I think we should format release date in locale @d-reader-josip - which locale?
3. Also I've discovered a couple of issues (not related to your work Ebi), when user creates comic issue and if he close form after first step, backend will return 500 error when tries to fetch comic issue, because some fields are null which we are expecting that are not null (covers for example). @d-reader-josip I don't know if this is something we should resolve asap?

**Comment by josip-fundl** [comment]
> @Agent-Ebimene it looks okay to me, but maybe let's do few changes here:
> 
>     1. Place release date field above authors field (create and edit screens)
> 
>     2. I think we should format release date in locale @d-reader-josip - which locale?

2. Unsure, I think our backend normalizes everything to ISO anyways?
3. at worst, we should put it in a Trello ticket, at best we'll resolve within 1-2 weeks

**Comment by luka-fundl** [comment]
> > @Agent-Ebimene it looks okay to me, but maybe let's do few changes here:
> > ```
> > 1. Place release date field above authors field (create and edit screens)
> > 
> > 2. I think we should format release date in locale @d-reader-josip - which locale?
> > ```
> 
> 2. Unsure, I think our backend normalizes everything to ISO anyways?
> 3. at worst, we should put it in a Trello ticket, at best we'll resolve within 1-2 weeks

@Agent-Ebimene 
```
{
		...request,
		releaseDate: request.releaseDate.toLocaleString(),
}
```
		
I think this should be our temp fix on Create/Edit comic issue endpoints

**Pull request merged**